integration_test/signatory.yaml

```diff
@@ -6,6 +6,14 @@ vaults:
     driver: file
     config:
       file: /etc/secret.json
+  aws:
+    driver: awskms
+    config:
+      user_name: stephen
+      access_key_id: AKIAWXBZ6RID2G4MXWIP
```

Comment:
@stephengaudet This is a security issue. Please remove this

Reply:
Pointed this out to Stephen earlier and deleted the API key